iOS Example Frame/iOS Example Frame/Helper/ApplePayCreator.swift

```diff
@@ -0,0 +1,53 @@
+import PassKit
+
+enum ApplePayCreator {
```

Comment:
File is inside the folder `Helper`. This is sample code, so maybe rename folder to `SampleCode`, and also move `ThemeDemo` alongside?

Reply:
The project is for for showing & testing our functionality. It contains parts common in projects (UI creations, overheads, placeholders, etc).


ThemeDemo and this ApplePayCreator are pieces of code that showcase using specific functionality to unlock more value out of our SDK. Might be clearer for discoverability if they were placed in a way that made them discoverable and easy to link to in documentation.